README.md

```diff
@@ -48,6 +48,12 @@ arguments. For example:
 WebServiceClient client = new WebServiceClient.Builder(6, "ABCD567890").build();
 ```
 
+If you would like to use the Sandbox environment, you can use the `host` method 
+that belongs to the Bulder. For example,
+```java
+WebServiceClient.Builder.host("sandbox.maxmind.com")
```

Comment:
Maybe this should be something like `WebServiceClient client = new WebServiceClient.Builder(6, "ABCD567890").host("sandbox.maxmind.com").build();`?